src/pages/home/report/ReportActionsView.js

```diff
@@ -46,8 +62,12 @@ class ReportActionsView extends React.Component {
         this.scrollToListBottom = this.scrollToListBottom.bind(this);
         this.recordMaxAction = this.recordMaxAction.bind(this);
         this.onVisibilityChange = this.onVisibilityChange.bind(this);
-        this.sortedReportActions = this.updateSortedReportActions();
+
+        this.sortedReportActions = [];
         this.timers = [];
+        this.unreadActionCount = 0;
+        this.shouldShowNewActionIndicator = true;
```

Comment:
Just noticing that this is the only place you refer to a `newActionIndicator` instead of an `unreadActionIndicator`, so we probably want to switch that for the sake of consistency.

Reply:
I have no idea where this `New` came from, it really makes no sense.